docs/extending/reusable_back_office_templates.md

```diff
@@ -0,0 +1,91 @@
+# Reusable Back Office templates
+
+When extending the Back Office, you can use base Twig templates for commonly used UI components such as tabs or tables.
+
+The available templates are:
+
+- `@ezdesign/ui/component/table/table.html.twig`
+- `@ezdesign/ui/component/tab/tabs.html.twig`
+
+To use the components, either [`embed`](https://twig.symfony.com/doc/3.x/tags/embed.html)
+or [`include`](https://twig.symfony.com/doc/3.x/tags/include.html) them in templates.
+
+Using `embed` enables you to override blocks defined inside the included template, for example:
+
+``` html+twig
+{% embed '@ezdesign/ui/component/table/table.html.twig' %}
+    {% block headline %}
+        Headline
+    {% endblock %}
+
+    {% block actions %}
+        <a href="#" class="btn btn-icon">
+            <svg class="ibexa-icon ibexa-icon--small ibexa-icon--create">
+                <use xlink:href="{{ ez_icon_path('create') }}"></use>
+            </svg>
+        </a>
+    {% endblock %}
+    
+    {% block thead %}
+        <tr>
+            <th></th>
+            <th>Column A</th>
+            <th>Column B</th>
+            <th>Column C</th>
+        </tr>
+    {% endblock %}
+
+    {% block tbody %}
+        {% for i in 1..10 %}
+            <tr>
+                <td></td>
+                <td>A{{ i }}</td>
+                <td>B{{ i }}</td>
+                <td>C{{ i }}</td>
+            </tr>
+        {% endfor %}
+    {% endblock %}
+{% endembed %}
+```
+
+Use `include` instead of `embed` when you pass tab content as variable while rendering the template:
+
+``` html+twig
+{% include '@ezdesign/ui/component/tab/tabs.html.twig' with {
+        tabs: [
+          { id: 'first', label: 'First', content: 'First tab content' },
+          { id: 'second', label: 'Second', content: 'Second tab content', active: true },
+        ]
+    } %}
+```
```

Comment:
That's what happens when you do code review on Monday.

Currently this alternative usage is only available for tabs component, so let's put usage example for each of these components, with embed explanation as part of introduction. Pretty please :pray: 

> ``` html+twig
>  {% embed '@ezdesign/ui/component/tab/tabs.html.twig' with {
>         tabs: [
>             { id: 'foo', label: 'Foo' },
>             { id: 'bar', label: 'Bar' },
>         ]
>     } %}
>         {% block tab_content %}
>             {% embed '@ezdesign/ui/component/tab/tab_pane.html.twig' with { id: 'foo', active: true } %}
>                 {% block content %}
>                     Foo tab
>                 {% endblock %}
>             {% endembed %}
> 
>             {% embed '@ezdesign/ui/component/tab/tab_pane.html.twig' with { id: 'bar' } %}
>                 {% block content %}
>                     Bar tab. <p>Some <b>Rich</b> HTML <a href="#">content</a></p>
>                 {% endblock %}
>             {% endembed %}
>         {% endblock %}
>     {% endembed %}
> ```
> 
> Alternatively you can use `include` instead of `embed` when you pass tab content as variable while rendering the template:
> 
> ``` html+twig
> {% include '@ezdesign/ui/component/tab/tabs.html.twig' with {
>         tabs: [
>           { id: 'first', label: 'First', content: 'First tab content' },
>           { id: 'second', label: 'Second', content: 'Second tab content', active: true },
>         ]
>  } %}
> ```